pkg/scheduler/factory.go

```diff
@@ -357,21 +357,21 @@ func MakeDefaultErrorFunc(client clientset.Interface, podLister corelisters.PodL
 		// Check if the Pod exists in informer cache.
 		cachedPod, err := podLister.Pods(pod.Namespace).Get(pod.Name)
 		if err != nil {
-			klog.InfoS("Pod doesn't exist in informer cache", "pod", klog.KObj(pod), "err", err)
-			return
+			return fmt.Errorf("pod doesn't exist in informer cache")
```

Comment:
Is the only error that the Lister report for not found? Otherwise, we can do

`fmt.Errorf("reloading pod from cache: %w", err)`

Reply:
How about `fmt.Errorf("obtaining pod from cache: %w", err)`?